sdk/resourcemanager/Azure.ResourceManager.Core/src/Operation/PhArmOperation.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+
+namespace Azure.ResourceManager.Core
+{
+    /// <summary>
+    /// A calss representing an arm operation wrapper object.
+    /// </summary>
+    /// <typeparam name="TOperations"> The <see cref="ResourceOperationsBase"/> to convert TModel into. </typeparam>
+    /// <typeparam name="TModel"> The model returned by existing Operation methods. </typeparam>
+    public class PhArmOperation<TOperations, TModel> : ArmOperation<TOperations>
```

Comment:
Is this called PhArmOperations because it will be removed in a future PR?

Reply:
yes this won't be needed once the generated code from Resources is moved into Core